api_grpc/worker.go

```diff
@@ -419,6 +421,7 @@ func NotifyWorkers() {
 			}
 			workerIndex := getWorkerWithLeastWorkload(workers)
 			workers[workerIndex].Workload += 3
+			workersForStream[streams[i].ID] = append(workersForStream[streams[i].ID], workers[workerIndex])
```

Comment:
This needs to be persisted in the database. Otherwise, streams can't be stopped if TUM Live is restarted during a stream.

Reply:
I only wanted to get it working right now, once it works on the test server I will adapt this.